src/schema/notifications.ts

```diff
@@ -302,8 +324,11 @@ export class ZimbraNotifications {
 		if (items) {
 			items.forEach((i: any) => {
 				const item = normalizeFolder(i);
+				const itemId = get(i, 'id', '').includes(':')
```

Comment:
We can access `item.id` here safely instead of using `get`, and it is confusing to use `i` instead of `item`.

Reply:
Done.